package.json

```diff
@@ -17,14 +17,22 @@
     "redux-thunk": "^2.2.0"
   },
   "devDependencies": {
+    "eslint-config-prettier": "^2.7.0",
+    "prettier": "^1.8.2",
     "react-scripts": "1.0.13"
   },
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test --env=jsdom",
     "watch": "node scripts/watch.js",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "format:check": "prettier -l ./src/**/*.js",
+    "format:fix": "prettier --write ./src/**/*.js",
+    "lint:check": "eslint ./src",
+    "lint:fix": "eslint --fix ./src && npm run format:fix",
```

Comment:
@Spetastian I see...  Idea: how about we change `format:fix` to just `format` and `lint:fix` to just `lint`?  I think contributors should only need to type 1 command to comply with our coding standards, and that it should be very brief (like `npm run fix` or `npm run lint`).